source/stylesheets/refills/_accordion-tabs-basic.scss

```diff
@@ -0,0 +1,67 @@
+.accordion-tabs-basic {
+  $medium-screen: em(640) !default;
+  $tab-mode: $medium-screen;
+
+  @include clearfix;
+  padding: 0;
+
+  @include media(max-width $tab-mode) {
+  // Overall styles
+  }
+
+  .tab-header-and-content {
+    list-style: none;
+
+    @include media($tab-mode) {
+      display: inline;
+    }
+
+    &:first-child .tab-link {
+    // Top link style in list mode
+
+      @include media(max-width $tab-mode) {
+      // First link style in tab mode
+      }
+    }
+
+    &:last-child .tab-link {
+    // Last link style in list mode
+
+      @include media(max-width $tab-mode) {
+      // Rightmost link style in tab mode
+      }
+    }
+
+    a.tab-link {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3